### PR TITLE
data: anuragts

### PR DESCRIPTION
**Pull request by anuragts**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5258"></a>

### CODE REVIEW

File: data/anuragts.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "name": "Anurag Sharma",
+    "type": "personal",
+    "bio": "I create and break thing on and for the Internet",
+    "avatar": "https://github.com/anuragts.png",
+
```
Comment (schmelto) [inline data/anuragts.json:6]:
```suggestion
```

File: data/anuragts.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+    "name": "Anurag Sharma",
+    "type": "personal",
+    "bio": "I create and break thing on and for the Internet",
+    "avatar": "https://github.com/anuragts.png",
```
Comment (priyankarpal) [inline data/anuragts.json:5]: currently, you don't need to add an avatar. Delete this line.

```suggestion
 
```

File: data/anuragts.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "icon": "FaGithub"
+      },
+      {
+        "group": "Socials Primary",
+        "name": "LinkedIn: Let's connect",
+        "url": "https://linkedin.com/in/thanuragdev",
+        "icon": "FaLinkedin"
+      }
+    ],
+    "milestones": [
```
Comment (priyankarpal) [inline data/anuragts.json:45]: Please add your milestones. I think you forgot to add.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/anuragts.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    "tags": [
+      "Open Source",
+      "Typescript",
+      "NextJS",
+      "Modern Stack"
+    ],
+    "socials": [
+      { "icon": "FaTwitter", "url": "https://twitter.com/thanuragdev" },
+      { "icon": "FaGithub", "url": "https://github.com/anuragts" },
+      { "icon": "FaYoutube", "url": "https://youtube.com/thanuragdev" }
```
Comment (priyankarpal) [inline data/anuragts.json:14]: Your YouTube link is not working please look into it.

**Review by priyankarpal — APPROVED** [review]
Looks good 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Pull request merged**